### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by anishasthana**

### CODE REVIEW

File: ml-pipelines/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# ML Pipelines
+
+ML Pipelines is the Open Data Hub's pipeline solution for data scientists. It is built on top of the upstream [kfp-tekton](https://github.com/kubeflow/kfp-tekton) project. The Open Data Hub community has a fork of this upstream under the opendatahub org.
+
+1. [opendatahub-io/kfp-tekton](https://github.com/opendatahub-io/kfp-tekton)
```
Comment (accorvin) [inline ml-pipelines/README.md:5]: This link shows up weird. IMO we should make the word "fork" above a link to this repo.

File: ml-pipelines/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# ML Pipelines
+
+ML Pipelines is the Open Data Hub's pipeline solution for data scientists. It is built on top of the upstream [kfp-tekton](https://github.com/kubeflow/kfp-tekton) project. The Open Data Hub community has a fork of this upstream under the opendatahub org.
```
Comment (accorvin) [inline ml-pipelines/README.md:3]: Do we want to mention kubeflow pipelines here?

File: ml-pipelines/README.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+1. [opendatahub-io/kfp-tekton](https://github.com/opendatahub-io/kfp-tekton)
+
+## Installation
+
+### Prerequisites
+
+1. The cluster needs to be OpenShift 4.9 or higher
+2. OpenShift Pipelines 1.7.2 or higher needs to be installed on the cluster
+3. The Open Data Hub operator needs to be installed
+4. Create a namespace for ML Pipelines to be installed in. ML Pipelines is installed in the `odh-applications` namespace.
```
Comment (accorvin) [inline ml-pipelines/README.md:14]: This line confuses me. Does the user need to create the odh-applications namespace?

Comment (anishasthana) [reply ml-pipelines/README.md:14]: Whoops, I accidentally ate the `By default`

**Review by gmfrasca — APPROVED** [review]

**Pull request merged**